packages/slice-machine/src/modules/modal/index.ts

```diff
@@ -34,13 +34,10 @@ export const modalReducer: Reducer<ModalStoreType, ModalActions> = (
 ) => {
   switch (action.type) {
     case getType(modalCloseCreator):
-      return {
-        ...state,
-        [action.payload.modalKey]: false,
-      };
+      return initialState;
```

Comment:
I'm not sure I agree with this, since we don't know if we'll want to close all modals together in future. 

But if we do this, we could remove all the functions in `useSliceMachineActions` for each specific closeModal and only have one `closeModals` that closes all of them

Reply:
Good point for closeModals. I checked with @Heyrows and we can go forward with this